### ISSUE TITLE
Changes of behaviour with simp:false

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3884]
Imported from SourceForge on 2024-07-08 20:15:54
Created by **[sangwinc](https://sourceforge.net/u/sangwinc/)** on 2015-11-11 09:06:30
Original: https://sourceforge.net/p/maxima/bugs/3046

---

I've complied Maxima 5.37.3 on Debian from source with no problems.  Thank you for this.

The STACK project has failing unit tests caused by a change in the behaviour of Maxima with simp:false as follows.   A minimal example is below.

The old version is

    Maxima 5.36.1 http://maxima.sourceforge.net
    using Lisp CLISP 2.49 (2010-07-07)
    Distributed under the GNU Public License. See the file COPYING.
    Dedicated to the memory of William Schelter.
    The function bug_report() provides bug reporting information.
    (%i1) display2d:false;
    (%o1) false
    (%i2) simp:false;
    (%o2) false
    (%i3) (3/6)*x;
    (%o3) (3/6)*x
    (%i4) 

The new version gives

    Maxima 5.37.3 http://maxima.sourceforge.net
    using Lisp CLISP 2.49 (2010-07-07)
    Distributed under the GNU Public License. See the file COPYING.
    Dedicated to the memory of William Schelter.
    The function bug_report() provides bug reporting information.
    (%i1) display2d:false;
    (%o1) false
    (%i2) simp:false;
    (%o2) false
    (%i3) (3/6)*x;
    (%o3) (3*x)/6

So, with simp:false we have a change of behaviour of

    (3/6)*x -> (3*x)/6

This matters because I want to filter out coefficients in polynomials and rational numbers in other expressions and decide if a student has entered numbers in "lowest terms".  I know this is a very subtle change, but it breaks the "lowest terms" test.  Can anything be done about this?

Thanks,
Chris

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 20:15:56
Created by **[sangwinc](https://sourceforge.net/u/sangwinc/)** on 2015-11-11 09:07:00
Original: https://sourceforge.net/p/maxima/bugs/3046/#2714

---

- Description has changed:

Diff:

~~~~

--- old
+++ new
@@ -1,6 +1,6 @@
 I've complied Maxima 5.37.3 on Debian from source with no problems.  Thank you for this.
 
-The STACK project has unit tests caused by a change in the behaviour of Maxima with simp:false as follows.   A minimal example is below.
+The STACK project has failing unit tests caused by a change in the behaviour of Maxima with simp:false as follows.   A minimal example is below.
 
 The old version is
 

~~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 20:15:59
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2015-11-11 19:06:58
Original: https://sourceforge.net/p/maxima/bugs/3046/#2cc6

---

I think I see how to fix up NFORMAT. Chris, can you send me your unit tests? I will include them in the Maxima test collection.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 20:16:03
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2015-12-04 00:45:24
Original: https://sourceforge.net/p/maxima/bugs/3046/#cd00

---

Fixed by commit 1a3bfbd. Closing this report.
